app/ConsoleWriter.php

```diff
@@ -28,10 +28,10 @@ public function logStep($message)
 
     public function exec(string $command)
```

Comment:
Maybe we can wrap it in spaces just once in the `labeledLine` method?

Reply:
Done